advanced/dashboard/sso.mdx

```diff
@@ -90,5 +91,41 @@ Use single sign-on to your dashboard via SAML and OIDC. If you use Okta or Googl
             </Step>
         </Steps>
     </Tab>
+</Tabs>
+
+## Microsoft Entra
 
-</Tabs>
\ No newline at end of file
+<Tabs>
+    <Tab title="SAML">
+        <Steps>
+            <Step title="Create an application">
+                Under `Enterprise applications`, click to create a new application.
+                Select "Create your own application" and choose "Integrate any other application you don't find in the gallery (Non-gallery)".
+            </Step>
+            <Step title="Configure SAML">
+                Navigate to the Single Sign-On setup page and select SAML. Under Basic SAML Configuration, enter the following:
+                * Identifier (Entity ID): The Audience URI provided by Mintlify
+                * Reply URL (Assertion Consumer Service URL): The ACS URL provided by Mintlify
+                
+                Leave the other values blank and click Save.
+            </Step>
+            <Step title="Configure Attributes & Claims">
+                Edit the Attributes & Claims section:
+                * Click on the Unique User Identifier (Name ID) under Required Claim
+                * Change the Source attribute to use `user.primaryauthoritativeemail`
+                * Under Additional claims, create the following claims:
```

Comment:
```suggestion
                1. Click on the Unique User Identifier (Name ID) under Required Claim
                2. Change the Source attribute to use `user.primaryauthoritativeemail`
                3. Under Additional claims, create the following claims:
```

Since these are sequential, let's go with an ordered list